tools/pythonpkg/tests/extensions/substrait/test_ibis_tpch.py

```diff
@@ -76,83 +76,83 @@ def execute(require, query_number):
     duck_con = TPCHBackend(duck_con=connection)
     tpch_execute_ibis_to_duck_query(duck_con,query_number)
 
-def test_query_substrait_01(require):
-    execute(require,1)
+# def test_query_substrait_01(require):
```

Comment:
Should this be commented out?

Reply:
Because of @gforsyth update yday, not anymore :-)